mojo-nifi/nifi-nar-bundles/nifi-daimojo-record-bundle/nifi-daimojo-record-processors/src/main/java/org/apache/nifi/processors/daimojo/record/ExecuteDaiMojoScoringPipeline.java

```
@@ -0,0 +1,417 @@
+			.description("If a FlowFile fails processing for any reason (for example, the FlowFile records cannot be parsed), it " +
+						 "will be routed to this relationship")
+			.build();
+	
+	public static final Relationship REL_ORIGINAL = new Relationship.Builder()
+			.name("original")
+			.description("The original FlowFile that was scored. If the FlowFile fails processing, nothing will be sent to this relationship")
+			.build();
+			
+	private final static List<PropertyDescriptor> properties;
+	private final static Set<Relationship> relationships;
+			
+	private ComponentLog logger;
+	
+	private MojoPipeline model;
+	
+	private String mojoPipelineUUID;
+	
+	static {
+		ArrayList<PropertyDescriptor> propertiesBuilder = new ArrayList<>();
+		propertiesBuilder.add(RECORD_READER);
+		propertiesBuilder.add(RECORD_WRITER);
+		propertiesBuilder.add(PIPELINE_MOJO_FILEPATH);
+		properties = Collections.unmodifiableList(propertiesBuilder);
+		
+		final Set<Relationship> relationshipsBuilder = new HashSet<>();
+		relationshipsBuilder.add(REL_SUCCESS);
+		relationshipsBuilder.add(REL_FAILURE);
+		relationshipsBuilder.add(REL_ORIGINAL);
+		relationships = Collections.unmodifiableSet(relationshipsBuilder);
+	}
+	
+	@Override
+	public Set<Relationship> getRelationships() {
+		return relationships;
+	}
+	
+	@Override
+	protected List<PropertyDescriptor> getSupportedPropertyDescriptors() {
+		return properties;
+	}
+	
+	/**
+	 * Gets called once when the processor is scheduled to run
+	 * @param context
+	 */
+	@OnScheduled
+	public void onScheduled(final ProcessContext context) {
+		final String pipelineMojoPath = context.getProperty(PIPELINE_MOJO_FILEPATH).getValue();
+
+			try {
+				model = MojoPipelineService.loadPipeline(new File(pipelineMojoPath));
+				
+				mojoPipelineUUID = "pipeline.mojo uuid " + model.getUuid();
+			} catch (LicenseException | IOException e) {
+				getLogger().error("Unable to load MOJO from pipelineMojoPath due to " + e.toString());
+			}
```

nit: Over indented.

Also, in addition to (or instead of) logging an error, you may want to not catch the exception, instead letting it bubble up back to whatever NiFi code calls this function.

According to the following, throwing an exception here will both present the NiFi user with an error and stop the rest of the processors functions from being called.  https://docs.cloudera.com/HDPDocuments/HDF3/HDF-3.5.1/adding-functionality-to-apache-nifi/content/onscheduled.html